packages/framework/fluid-framework/api-report/fluid-framework.api.md

```diff
@@ -64,6 +24,194 @@ export enum AttachState {
     Detached = "Detached"
 }
 
+// @alpha
+export interface AttributionInfo {
+    timestamp: number;
+    user: IUser;
+}
+
+// @alpha
+export type AttributionKey = OpAttributionKey | DetachedAttributionKey | LocalAttributionKey;
+
+// @alpha @sealed
+export interface AttributionPolicy {
+    attach: (client: Client) => void;
+    detach: () => void;
+    // (undocumented)
```

Comment:
Probably all these should be documented too - obv a lot so not in this PR necessarily, but we should open some work items and consider them for 2.0.

Reply:
I see this PR as a way to make these issues more discoverable, so I think we should specifically not block it on trying to fix these issues, and instead merge this PR before farming out work to fix issues like this.